jinishakheni/BrainBash-Frontend
# Set up web navigation with login register

This merge request fixes multiple issues and problems. for more check commit messages.

Mainly:
Syntax to enable login modal popup is -> `toggleAuthForms("login", true);` can be access through ```const {toggleAuthForms } =
    useAuthFormsContext();``` . Closing the modal can be by close button, clicking out side of form, and by login.
    

Test:

- Check popup model for login and signup,
- Signup and login through popup modal form. On login it should close the form modal
- Signup and login through routes -> account/login and account/signup. Check links changes between login and register form
- Try forgot password through account/login form
- Try forgot password through account/register form
- Login as valid user and try the routes again, should navigate to home page
- Try forgot-password route, Also, it shouldn't show back to login button


close #23
